wp-content/plugins/jetpack/_inc/build/custom-css/custom-css/js/core-customizer-css.min.js

```diff
@@ -0,0 +1 @@
+!function(){var t,s;window.wp,t=jQuery,(s=window.wp.customize).controlConstructor.jetpackCss=s.Control.extend({modes:{default:"text/css",less:"text/x-less",sass:"text/x-scss"},_updating:!1,ready:function(){this.opts=window._jp_css_settings,this.$input=t("<textarea />",{name:this.setting.id,class:"for-codemirror hidden"}).val(this.setting()),this.container.append(this.$input),s(this.setting.id,_.bind((function(t){var i=new s.Element(this.$input);this.elements=[i],i.sync(t),i.set(t())}),this)),this.opts.useRichEditor?this.initCodeMirror():this.$input.removeClass("hidden"),s.bind("ready",_.bind(this.addLabels,this))},initCodeMirror:function(){this.editor=window.CodeMirror.fromTextArea(this.$input.get(0),{mode:this.getMode(),lineNumbers:!0,tabSize:2,indentWithTabs:!0,lineWrapping:!0}),this.addListeners()},addListeners:function(){var s=!1;t("#accordion-section-custom_css > .accordion-section-title").click(_.bind(_.debounce(this.editor.refresh,250),this.editor)),this.editor.on("focus",(function(t){t.refresh()})),this.editor.on("change",_.bind((function(t){this._updating=!0,this.$input.val(t.getValue()).trigger("change"),this._updating=!1,s||(window.ga&&window.ga("send","event","Customizer","Typed Custom CSS"),s=!0)}),this)),this.editor.on("focus",(function(){window.ga&&window.ga("send","event","Customizer","Focused CSS Editor")})),this.setting.bind("change",_.bind(this.externalChange,this))},getMode:function(){var t=s("jetpack_custom_css[preprocessor]")();return""!==t&&this.modes[t]||(t="default"),this.modes[t]},externalChange:function(){this._updating||this.editor.setValue(this.setting())},refresh:function(t){"accordion-section-custom_css"===t&&setTimeout(_.bind((function(){this.editor.refresh()}),this),300)},addLabels:function(){this.addTitle("jetpack_css_mode_control",this.opts.l10n.mode),this.addTitle("jetpack_mobile_css_control",this.opts.l10n.mobile),this.addDesc("wpcom_custom_css_content_width_control",this.opts.l10n.contentWidth);var s=this._getControl("wpcom_custom_css_content_width_control");s&&s.find("input").after("<span>px</span>"),t("<div />",{id:"css-help-links",class:"css-help"}).appendTo(this.container),t("<a />",{id:"help-link",target:"_blank",href:this.opts.cssHelpUrl,text:this.opts.l10n.css_help_title}).prependTo("#css-help-links"),this.opts.areThereCssRevisions&&t("<a />",{id:"revisions-link",target:"_blank",href:this.opts.revisionsUrl,text:this.opts.l10n.revisions}).prependTo("#css-help-links")},addTitle:function(t,s){var i=this._getControl(t);i&&i.prepend('<span class="customize-control-title">'+s+"<span>")},addDesc:function(t,s){var i=this._getControl(t);i&&i.append('<span class="description">'+s+"<span>")},_getControl:function(t){var i=s.control.value(t);return i?i.container:null}})}();
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).